joint_symbolic.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+import itertools
+from scipy.signal import find_peaks
+
+plt.style.use('seaborn-v0_8-ticks')
+
+
+def get_peaks(signal):
+    peaks, _ = find_peaks(signal, distance=115)
+    difference = np.diff(peaks)
+    return peaks, signal[peaks], difference
+
+
+def make_empty_matrix():
+    my_permutations = [''.join(map(str, triplet)) for triplet in list((itertools.product([1, 0], repeat=3)))]
+    my_matrix = pd.DataFrame()
+    for triplet_1 in my_permutations:
+        for triplet_2 in my_permutations:
+            my_matrix.at[
+                "'" + triplet_1, "'" + triplet_2] = 0  # dash needed so that R does not remove the initial zeros
+    return my_matrix
+
+
+def get_diff_df(dataframe, is_fv, is_left, is_ekg):
```

Comment:
i dwa: w opublikowanej na ZENODO bazie bedzie tylko jedno FV - to jest Twoje robocze rozwiazanie, ale publikowac go nie trzeba
nie piszemy też w artykule nic o ECG - wiec moze ten fragmnet ominac albo zmienic - pamietajac ze w bazie bedzie tylko 1 kolumna o nazwie "fv[cm/s]"